lua/autorun/jmod_legacy_server.lua

```diff
@@ -126,8 +126,13 @@ if(SERVER)then
 		ply.OriginalJumpPower=ply:GetJumpPower()
 		ply:SetJumpPower(0)
 		sent:SetDTBool(2,false)
-		ply:Give("wep_jack_sentrycontrols")
-		ply:SelectWeapon("wep_jack_sentrycontrols")
+		if ply:HasWeapon("wep_jack_gmod_hands") then
+			ply:Give("wep_jack_gmod_hands")
+			ply:SelectWeapon("wep_jack_gmod_hands")
+		else
+			ply:Give("wep_jack_sentrycontrols")
+			ply:SelectWeapon("wep_jack_sentrycontrols")
+		end
```

Comment:
can you explain this one pls

Reply:
I wanted to make it compatible with JMod but i remembered that JMod is already needed for this addon and i forgot to remove it from code :P

Reply:
Might remove it if it's not necessary then